packages/@react-spectrum/dialog/src/Dialog.tsx

```diff
@@ -22,6 +22,10 @@ import {SpectrumBaseDialogProps, SpectrumDialogProps} from '@react-types/dialog'
 import styles from '@adobe/spectrum-css-temp/components/dialog/vars.css';
 import {useDialog, useModalDialog} from '@react-aria/dialog';
 
+/**
+ * Dialogs display important information that users need to acknowledge.
+ * They appear over the interface and block further interactions.
```

Comment:
Note to self, adjust this description to reflect the split of AlertDialog from Dialog aka change the "display important information..." line